collect_app/src/main/java/org/odk/collect/android/widgets/SelectMultiWidget.java

```diff
@@ -132,6 +140,15 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
                         buttonView.setChecked(true);
                     }
                 }
+
+                // show warning when selected choice value has spaces
+                int index = (int) checkBox.getTag();
+                String value = items.get(index).getValue();
+                if (isChecked && value != null && value.contains(" ")) {
+
```

Comment:
Do you want this blank line?

Reply:
Yes, it does help with readability

Reply:
I used to put a blank line *above* all my `if`s and loops.